internal/pkg/cli/pipeline_init.go

```diff
@@ -23,7 +26,10 @@ const (
 	pipelineAddEnvPrompt          = "Would you like to add an environment to your pipeline?"
 	pipelineSelectEnvPrompt       = "Which environment would you like to add to your pipeline?"
 	pipelineEnterGitHubRepoPrompt = "What is your application's GitHub repository?" // TODO allow just <user>/<repo>?
-	pipelineDefaultFilename       = "pipeline.yml"
+)
+
+const (
+	buildspecTemplatePath = "cicd/buildspec.yml"
 )
```

Comment:
Is there a reason for a separate `const` block?

Reply:
I've been clustering constants into separate blocks kindof like here when they're related: https://golang.org/src/math/const.go

In this situation, the first block is for prompts whereas this one is to hold paths.